dataverse-webapp/src/main/java/edu/harvard/iq/dataverse/ingest/IngestServiceBean.java

```diff
@@ -357,42 +360,29 @@ public void startIngestJobsForDataset(Dataset dataset, AuthenticatedUser user) {
         startIngestJobs(scheduledFiles, user);
     }
 
-    public String startIngestJobs(List<DataFile> dataFiles, AuthenticatedUser user) {
+    public StartIngestResult startIngestJobs(List<DataFile> dataFiles, AuthenticatedUser user) {
+        Preconditions.checkState(dataFiles.stream().allMatch(DataFile::isIngestScheduled), "DataFile(s) must be scheduled for ingest to queue them for ingesting");
```

Comment:
[Minor] Our coding conventions state: _try not to excees 160 chars_. Not entirely sure if that rule applies to this repository or not, but it will just read easier if we add a line break.

```suggestion
        Preconditions.checkState(dataFiles.stream().allMatch(DataFile::isIngestScheduled),
                         "DataFile(s) must be scheduled for ingest to queue them for ingesting");
```

And there's another one in line 382.